frontend/workflows/experimentation/src/list-experiments.tsx

```diff
@@ -74,12 +74,14 @@ interface ListExperimentsProps {
 }
 
 const ListExperiments: React.FC<ListExperimentsProps> = ({ columns, experimentTypes }) => {
-  const [experiments, setExperiments] = useState([]);
+  const [experiments, setExperiments] = useState<
+    IClutch.chaos.experimentation.v1.Experiment[] | undefined
+  >(undefined);
   const [error, setError] = useState("");
 
   const navigate = useNavigate();
 
-  if (experiments.length === 0) {
+  if (experiments === undefined) {
```

Comment:
If you only want to call this once on page load you could use a react hook

```jsx
React.useEffect(() => {
  client....
}, []);
```

Reply:
nice! fixed - thanks!